read-the-docs-2017-stats.rst

```diff
@@ -0,0 +1,109 @@
+.. post:: Jan 26, 2018
+   :tags: stats, year-in-review
+
+Read the Docs 2017 Stats
+========================
+
+2017 has been a good year for Read the Docs.
```

Comment:
"has been" might be the wrong tense here, implying its still 2017. "was" avoids this.